.github/workflows/release_portable_linux_pytorch_wheels.yml

```diff
@@ -87,7 +87,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python_version: ["3.11", "3.12", "3.13"]
+        python_version: ["3.10", "3.11", "3.12", "3.13"]
```

Comment:
This will increase our testing surface area. I've started some discussions about whether we always want to run tests on all python versions or if we can choose a subset of python versions to run (regular) tests for.

It's about 20-120 minutes of test time per python version, per GPU family, per PyTorch version. That's _with our current set of tests_, and we might expand the set of tests later.